src/main/java/se/kth/sda/skeleton/user/UserService.java

```diff
@@ -12,14 +13,39 @@
     @Autowired
     private BCryptPasswordEncoder passwordEncoder;
 
-
     public User findUserByEmail(String email) {
         return userRepository.findByEmail(email);
     }
 
+    /**
+     * check if the accountType is valid
+     *
+     * @param user user who's accountType is checked
+     * @return true if accountType is valid, false otherwise
+     */
```

Comment:
Tiny amendments:

- Capitalise 'C'
- Amend to 'whose'

```
/**
     * Check if the accountType is valid
     *
     * @param user user whose accountType is checked
     * @return true if accountType is valid, false otherwise
     */
```